doc/cephfs/cephfs-shell.rst

```diff
@@ -3,16 +3,22 @@
 CephFS Shell
 =============
 
-The File System (FS) shell includes various shell-like commands that directly interact with the :term:`Ceph File System`.
+CephFS Shell provides shell-like commands that directly interact with the
+:term:`Ceph File System`.
+
+Behaviour of CephFS Shell can be tweaked using cephfs-shell.conf. CephFS Shell
+looks for it by default at the path provided in environment variable
+`CEPHFS_SHELL_CONF`, then in current working directory and then in user's home
+directory.
```

Comment:
give the name of that file in the home directory

Reply:
I am not sure what you mean but I have updated that line. Please take a look and let me know - https://github.com/ceph/ceph/pull/33286/files?file-filters%5B%5D=.py&file-filters%5B%5D=.rst&file-filters%5B%5D=No+extension#diff-f762f1d9ff1b1e7af6492d399e35b748R12